TestPullRequest/src/testPackage/SubCalc.java

```diff
@@ -2,6 +2,7 @@
 
 public class SubCalc extends Calc {
 	
+	@Override
 	public Integer addition(int a, int b, int c) {
```

Comment:
@LHommeDeBot Rename this method to addieren. For more information check github.com